tests/python/gaia-ui-tests/gaiatest/apps/contacts/regions/contact_details.py

```diff
@@ -94,11 +94,9 @@ def a11y_click_edit(self):
         return EditContact(self.marionette)
 
     def tap_back(self):
-        el = self.marionette.find_element(*self._details_header_locator)
-        Wait(self.marionette).until(expected.element_displayed(el))
-        # TODO: remove tap with coordinates after Bug 1061698 is fixed
-        el.tap(25, 25)
-        Wait(self.marionette).until(expected.element_not_displayed(el))
+        back_button = self.marionette.find_element(*self._details_header_locator)
+        self.tap_element_from_system_app(element=back_button)
+        Wait(self.marionette).until(expected.element_not_displayed(back_button))
```

Comment:
No, it's used when the button that's being tapped upon closes the app. When that happens, Marionette is out of whack, because it still thinks the active frame that is being worked on is that app. See https://bugzilla.mozilla.org/show_bug.cgi?id=1164078 for more explanation.
For shadow DOM elements, you just have to use coordinates.